src/designer/elsa-workflows-studio/src/components/designers/elsa-designer/elsa-designer.tsx

```diff
@@ -0,0 +1,1151 @@
+import {Component, Event, EventEmitter, h, Host, Method, Prop, State, Watch, Listen } from '@stencil/core';
+import {v4 as uuid} from 'uuid';
+import {
+  addConnection,
+  findActivity,
+  getChildActivities,
+  getInboundConnections,
+  getOutboundConnections,
+  Map,
+  removeActivity,
+  removeConnection
+} from '../../../utils/utils';
+import {
+  ActivityDescriptor,
+  ActivityDesignDisplayContext,
+  ActivityModel,
+  ActivityTraits,
+  ConnectionModel,
+  EventTypes,
+  WorkflowModel,
+  WorkflowPersistenceBehavior,
+  ActivityDefinitions,
+
+} from '../../../models';
+import {eventBus} from '../../../services';
+import state from '../../../utils/store';
+import {ActivityIcon} from '../../icons/activity-icon';
+import {ActivityContextMenuState, LayoutDirection, WorkflowDesignerMode, LayoutState} from "../tree/elsa-designer-tree/models";
+
+import dagre from "dagre";
+import {JsPlumbUtils} from "./jsplumb-utils";
+import {
+  Connection as JsPlumbConnection,
+  DragEventCallbackOptions,
+  Endpoint,
+  EndpointOptions,
+  jsPlumbInstance
+} from "jsplumb";
+
+@Component({
+  tag: 'elsa-designer',
```

Comment:
The reason the original designer component was called `elsa-designer-tree` is to indicate its nature (a designer using a tree-like structure).

Since this new component is about drag and drop, how about renaming the designer to e.g. `elsa-designer-dragdrop`?

Or perhaps better yet: `elsa-designer--jsplumb`, so that we can potentially add additional designer implementations like `elsa-designer-x6`. 

The goal is for the designer to be pluggable and letting the developer choose the right designer for their needs.